ui/kotlinx-coroutines-android/src/AndroidExceptionPreHandler.kt

```diff
@@ -4,30 +4,42 @@
 
 package kotlinx.coroutines.android
 
+import android.os.*
 import android.support.annotation.*
 import kotlinx.coroutines.*
 import java.lang.reflect.*
 import kotlin.coroutines.*
 
-private val getter =
-    try {
-        Thread::class.java.getDeclaredMethod("getUncaughtExceptionPreHandler").takeIf {
-            Modifier.isPublic(it.modifiers) && Modifier.isStatic(it.modifiers)
-        }
-    }
-    catch (e: Throwable) { null /* not found */ }
+private val getter by lazy {
```

Comment:
Shouldn't this be defined inside `AndroidExceptionPreHandler` to avoid a second class being generated for the file members.

Reply:
Good point, though moving it inside AEPH is not enough